### ISSUE TITLE
Test set data

### ISSUE THREAD

**Issue by CoderX-820** [tiuxuxsh76075/MSWAL-#4]
Hello, I'd like to ask where I can find the test set data

**Comment by wuzhaodongaipython** [comment]
Hi,thank you for attentioning MSWAL.
We do not plan to release the test set data recently.
Maybe you can use training set and split it with 7:3 for a new dataset.
Thank you!
